routes/web.php

```diff
@@ -184,6 +184,7 @@
         Route::get('', "UserApiController@getAll");
         Route::post('', ['middleware' => ['openstackid.currentuser.serveradmin.json'], 'uses' => "UserApiController@create"]);
         Route::put('me', "UserApiController@updateMe");
+        Route::get('user_actions', ['middleware' => ['openstackid.currentuser.serveradmin.json'], 'uses' => "UserActionApiController@getActions"]);
```

Comment:
use of this middle-ware is incorrect
u just use just "auth" middle ware

Reply:
done